html/about.html

```diff
@@ -35,7 +40,7 @@ <h5 class="creative-logo-title">Creative Design & Content 2018 Logo</h5>
       <p class="creative-description">The logo of Design was decided through the logo competition</p>
       <a class="logo" href="../index.html">Design & Content</a>
     </section>
-    <section class="past-events">
+    <section class="past-events" id="events">
```

Comment:
- [ ] In this section for the desktop version the pictures should be horizontal:

Yours:
![Screenshot from 2021-11-16 18-05-56](https://user-images.githubusercontent.com/62705947/142031901-c8b85c83-fb44-45e4-9728-c5db3e14a65f.png)
Original:
![Screenshot from 2021-11-16 18-06-23](https://user-images.githubusercontent.com/62705947/142031926-88d2dd95-891d-4a00-8c9a-75a00d656096.png)

Reply:
done !!